sourcecode/www/src/components/ResourceModal/ResourceStats.jsx

```diff
@@ -8,71 +8,100 @@ import {
     XAxis,
     YAxis,
 } from 'recharts';
-import { fillEmptyDays } from '../../helpers/chart.js';
```

Comment:
Unrelated code style changes shouldn't be submitted. They make it harder to perform code reviews, and the original code ends up being misattributed (e.g. by `git blame`) to your commit.

PHPStorm can reformat portions of code. You should reformat only the stuff you added or changed.